website/docs/cloud-docs/workspaces/settings/drift-detection.mdx

```diff
@@ -0,0 +1,53 @@
+1. Select **Enable** under **Drift Detection**.
+1. Click **Save settings**.
+
+Terraform Cloud starts drift detection immediately if there no active Terraform runs in the workspace. If you enable drift detection during a speculative plan, Terraform Cloud will start drift detection soon after completion. If you enable drift detection during other types of runs, Terraform Cloud will start the first automatic drift detection job in about 24 hours.
+
+Drift detection never interrupts or interferes with runs. Terraform Cloud begins drift detection assessments when at least 24 hours have passed since the last assessment and the workspace does not have a run in progress. You can also start new runs during a drift detection assessment. Starting a new run cancels the current assessment, and Terraform Cloud automatically starts the next one in 24 hours.
+
+Drift detection runs in the background and does not affect your concurrency limit. Terraform Cloud also monitors and controls drift detection concurrency, so it will not cause issues for large-scale deployments with thousands of workspaces. However, drift detection for organizations with a large number of workspaces may take longer to complete.
+
+## Workspace Drift Status
+
+After you enable drift detection, the workspace displays a drift status. A workspace can have one of the following drift statuses:
+
+- **Pending:** Terraform Cloud is performing a drift detection assessment for the workspace or there are no prior drift detection results. For example, a workspace will show this status when you enable drift detection during an active run.
+- **Drift:** Terraform Cloud has found that one or more real-world infrastructure resources do not match the workspace’s state file.
+-  **Drift Check Failed:** Terraform Cloud could not successfully complete the drift detection assessment for this workspace and will retry in 24 hours.
+- **Checked:** Terraform Cloud successfully completed the drift detection assessment for this workspace and found no drift.
+
+The workspace’s **Drift** tab contains details about the last drift detection assessment. If the workspace has drifted, Terraform Cloud displays a log file showing how the real-world infrastructure has changed from the latest version of the workspace’s state file.
+
+
+## Correcting Drift
+
+When Terraform Cloud finds drift in a workspace, you can use one of the following approaches to correct that drift:
+
+- **Overwrite Drift:** Queue a new plan to realign your real-world infrastructure with your Terraform configuration.
+- **Update Terraform state and configuration to match drift:** Queue a [refresh-only plan](/cloud-docs/run/modes-and-options#refresh-only-mode) to update your Terraform state to match your real-world infrastructure. You can also modify your Terraform configuration to include any new or changed resources. Refer to our [Manage Resource Drift](https://learn.hashicorp.com/tutorials/terraform/resource-drift) tutorial for a detailed example.
```

Comment:
I think we should add a note here reminding folks that if they use the refresh-only strategy **without updating their configuration**, drift will no longer be detected until they run their next apply (that would revert the accepted changes to their state file).